src/utils/api.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+const ncNewsAPI = axios.create({ baseURL: 'https://nc-news-452q.onrender.com/api' });
+
+export function getTopics()
+{
+    return ncNewsAPI.get('/topics')
+        .then(({ data }) =>
+        {
+            return data.topics;
+        });
+}
+
+export function getArticles(topic)
+{
+    const query = topic === 'all' ? '' : `?topic=${topic}`;
+    return ncNewsAPI.get(`/articles${query}`)
+        .then(({ data }) =>
+        {
+            return data.articles;
+        });
+}
```

Comment:
Axios get methods can take query params as the second  argument instead of updating the url based on the query variables. 
`.get("/",{params"{topic}}).then()`